src/main/java/net/earthcomputer/clientcommands/command/ChorusCommand.java

```diff
@@ -0,0 +1,120 @@
+        return argument("pos", vec3())
+                .executes(ctx -> setGoal(
+                    getVec3(ctx, "pos").floorAlongAxes(EnumSet.allOf(Direction.Axis.class)),
+                    getVec3(ctx, "pos").add(1, 2, 1).floorAlongAxes(EnumSet.allOf(Direction.Axis.class)), relative))
+                .then(literal("--perfectly")
+                    .executes(ctx -> setGoal(
+                        getVec3(ctx, "pos").floorAlongAxes(EnumSet.allOf(Direction.Axis.class)).add(0.3, 0, 0.3),
+                        getVec3(ctx, "pos").floorAlongAxes(EnumSet.allOf(Direction.Axis.class)).add(0.7, 2, 0.7), relative)));
+    }
+
+    public static int setGoal(Vec3d v1, Vec3d v2, boolean relative) {
+        if (relative &&
+            (Math.abs(v1.getX()) > 8.0 || Math.abs(v1.getY()) > 8.0 || Math.abs(v1.getZ()) > 8.0) &&
+            (Math.abs(v2.getX()) > 8.0 || Math.abs(v2.getY()) > 8.0 || Math.abs(v2.getZ()) > 8.0)) {
+            sendError(new LiteralText("Goal is to far away!"));
+            return -1;
+        }
+        TempRules.chorusGoalV1 = v1;
+        TempRules.chorusGoalV2 = v2;
+        TempRules.chorusRelativeTel = relative;
+        sendFeedback("Set " +
+                (relative ? "relative" : "absolute")
+                + " goal area from " + TempRules.chorusGoalV1 + " to " + TempRules.chorusGoalV2);
+        return 0;
+    }
+
+    public static boolean onEat(Vec3d pos, int particleCount, int itemUseTimeLeft) {
+        Vec3d tempGoalV1;
+        Vec3d tempGoalV2;
+
+        if (TempRules.chorusRelativeTel) {
+            tempGoalV1 = TempRules.chorusGoalV1.add(pos);
+            tempGoalV2 = TempRules.chorusGoalV2.add(pos);
+        } else {
+            //Check if goal is even in range
+            tempGoalV1 = TempRules.chorusGoalV1.subtract(pos);
+            tempGoalV2 = TempRules.chorusGoalV2.subtract(pos);
+            if (!(Math.abs(tempGoalV1.getX()) > 8.0 || Math.abs(tempGoalV1.getY()) > 8.0 || Math.abs(tempGoalV1.getZ()) > 8.0) ||
+                !(Math.abs(tempGoalV2.getX()) > 8.0 || Math.abs(tempGoalV2.getY()) > 8.0 || Math.abs(tempGoalV2.getZ()) > 8.0)) {
+                tempGoalV1 = TempRules.chorusGoalV1;
+                tempGoalV2 = TempRules.chorusGoalV2;
+            } else {
+                sendError(new LiteralText("Goal is to far away!"));
+                return false;
+            }
+        }
+
+        final Vec3d GoalV1 = new Vec3d(Math.min(tempGoalV1.getX(), tempGoalV2.getX()), Math.min(tempGoalV1.getY(), tempGoalV2.getY()), Math.min(tempGoalV1.getZ(), tempGoalV2.getZ()));
+        final Vec3d GoalV2 = new Vec3d(Math.max(tempGoalV1.getX(), tempGoalV2.getX()), Math.max(tempGoalV1.getY(), tempGoalV2.getY()), Math.max(tempGoalV1.getZ(), tempGoalV2.getZ()));
+
+        return throwItemsUntil(rand -> {
+            if (particleCount != 16)
```

Comment:
All if statements should have braces after them. There is some old code in clientcommands that doesn't follow this rule, but all new code should adhere to this.